### PR TITLE
A nebadr/add roles to luis authoring

### PR DESCRIPTION
**Pull request by nebadr**
Add roles to luis authoring

### CODE REVIEW

**Comment by nebadr** [comment]
@yangyuan  Please review

**Review by yangyuan — CHANGES_REQUESTED** [review]
Missing version bump/release notes

**Comment by yangyuan** [comment]
Hi @nebadr 
For future .NET sdk PRs: As long as the PR doesn't contain customization codes, my review will be optional.

**Comment by nebadr** [comment]
@yangyuan Added the release version and notes

**Review by yangyuan — APPROVED** [review]

**Comment by nebadr** [comment]
@dsgouda Please review the PR.

**Comment by shahabhijeet** [comment]
@AlexGhiondea do we have anyone who can take a look at this PR

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/IModel.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public partial interface IModel
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        /// <exception cref="Microsoft.Rest.HttpOperationException">
+        /// <exception cref="ErrorResponseException">
```
Comment (AlexGhiondea) [inline sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/IModel.cs]: Is this a breaking change or just a fix to the documentation?

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj [hunk 2]
```diff
@@ -2,19 +2,14 @@
...
     This is a public release of the Cognitive Services LUIS Authoring SDK.
 
     Changes in this release:
-    1. Add packaging APIs to export applications for the runtime container. For more info, check out our container documentation: https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-container-howto
-    2. Add app version settings APIs.
-    3. Add azure account APIs to programmatically assign, get and remove azure subscriptions to a LUIS application. Previously, this was only supported via the LUIS portal.
-    4. Remove region parameter from publish, LUIS now automatically publishes to the regions in which the user has assigned Azure keys.
-    5. Add failed regions in partial success response of the publish API.
-    5. Fix some spelling mistakes in the API documentation.
+    1. Supporting Roles
```
Comment (AlexGhiondea) [inline sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj]: Could you add more information here?

**Review by AlexGhiondea — APPROVED** [review]
I left 2 comments, but looks good otherwise.

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/IModel.cs (hunk 1 above)
Comment (nebadr) [reply sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/IModel.cs]: I'll revert this change for now and just support the roles in this PR

**Review by AlexGhiondea — APPROVED** [review]

**Pull request merged**